specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-preview.1/TextAnalytics.json

```diff
@@ -77,10 +77,16 @@
               "$ref": "#/definitions/EntitiesResult"
             }
           },
-          "default": {
```

Comment:
You probably want to keep the default (e.g. what are you returning on a 429 response?)

Reply:
Sure I will add those

Reply:
Added